.github/workflows/build-push.yml

```diff
@@ -0,0 +1,167 @@
+name: Build and upload to PyPi
+
+on: [push, pull_request]
+
+
+env:
+ CIBW_TEST_COMMAND_LINUX: "pytest --import-mode append {project}/tests/unit -k 'not (test_connection_initialization or test_cloud)' && EVENT_LOOP_MANAGER=gevent pytest --import-mode append {project}/tests/unit/io/test_geventreactor.py && EVENT_LOOP_MANAGER=eventlet pytest --import-mode append {project}/tests/unit/io/test_eventletreactor.py "
+ CIBW_TEST_COMMAND_MACOS: "pytest --import-mode append {project}/tests/unit -k 'not (test_multi_timer_validation or test_empty_connections or test_connection_initialization or test_timer_cancellation or test_cloud)' "
+ CIBW_TEST_COMMAND_WINDOWS: "pytest --import-mode append {project}/tests/unit  -k \"not (test_deserialize_date_range_year or test_datetype or test_libevreactor or test_connection_initialization or test_cloud)\" "
+ CIBW_BEFORE_TEST: "pip install -r {project}/test-requirements.txt pytest"
+ CIBW_BEFORE_BUILD_LINUX: "rm -rf ~/.pyxbld && yum install -y epel-release && yum install -y redhat-rpm-config gcc libffi-devel python-devel libev libev-devel openssl openssl-devel"
+ CIBW_ENVIRONMENT: "CASS_DRIVER_BUILD_CONCURRENCY=2"
+ CIBW_SKIP: cp35*
+
+jobs:
+  build_wheels:
+    name: Build wheels ${{ matrix.os }} (${{ matrix.platform }})
+    if: contains(github.event.pull_request.labels.*.name, 'test-build') || github.event_name == 'push' && endsWith(github.event.ref, 'scylla')
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            platform: x86_64
+
+          - os: ubuntu-18.04
+            platform: i686
+
+          - os: ubuntu-18.04
+            platform: PyPy
+
+          - os: windows-latest
+            platform: win32
+
+          - os: windows-latest
+            platform: win64
+
+          - os: windows-latest
+            platform: PyPy
+
+          - os: macos-latest
+            platform: all
+
+          - os: macos-latest
+            platform: PyPy
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.7.1
+
+      - name: Install Visual C++ for Python 2.7
```

Comment:
We still supported `Pyhon2.7`?

Reply:
yes we do for the `python-driver` we publish a wheel for that too